index.js

```diff
@@ -41,20 +44,22 @@ var isFinished = onFinished.isFinished
  */
 
 function createHtmlDocument (message) {
-  var body = escapeHtml(message)
```

Comment:
Requesting help to understand, do we really need change this? Perhaps it is already handling multiple line error message?

Reply:
Also just to chime in, this kind of thing is exactly why (as requested in @IamLizu's other comment) we try to avoid unnecessary formatting changes. It makes it harder to figure out what is a functional change needing attention and what is purely presentation and can be ignored.